pkg/subscraping/sources/commoncrawl/commoncrawl.go

```diff
@@ -103,11 +104,10 @@ func (s *Source) getSubdomains(ctx context.Context, url string, domain string, s
 			}
 			resp.Body.Close()
 
-			src := string(body)
+			src, _ := string(body)
 
 			for _, subdomain := range session.Extractor.FindAllString(src, -1) {
 				subdomain = strings.TrimPrefix(subdomain, "25")
```

Comment:
I did not remove this line because I don't know what is for. Does anyone know what char is pretending to remove?